conf/application.conf

```diff
@@ -45,8 +45,6 @@ play.modules.enabled += "uk.gov.hmrc.play.bootstrap.backend.BackendModule"
 
 play.modules.enabled += "uk.gov.hmrc.agentclientrelationships.module.RecoverySchedulerModule"
 
-play.http.filters = "uk.gov.hmrc.play.bootstrap.backend.filters.BackendFilters"
```

Comment:
Approving on the basis that `BackendFilters` appear to be deprecated (https://github.com/hmrc/bootstrap-play/blob/843085646dce148f1a223ca6090e01b6e37fc9b2/bootstrap-backend-play-26/src/main/scala/uk/gov/hmrc/play/bootstrap/package.scala#L45)